pkg/plugins/plugindef/plugindef.cue

```diff
@@ -120,6 +120,24 @@ seqs: [
 					...
 				}
 
+				#ExtensionsLink: {
+					// Unique identifier of the plugin link extension
+					id:          string
+					description: string
+					// Relative path to the plugin
+					path: string
+					...
+				}
+
+				// A extension point provided by the plugin.
```

Comment:
"provided" sounds to me like this is where these extension points are defined, not where an extension point defined elsewhere is implemented. AIUI, we're in the latter world - the set of extension points is statically defined elsewhere in core

Reply:
So this is a link that is exposed from the plugin to Grafana core or other plugins. So in this case the plugin is providing the extension point.